site/docs/components/highcharts-theme/examples.mdx

```diff
@@ -78,6 +78,18 @@ Use a bullet chart to show performance against a target across categories. Each
 
 <LivePreview componentName="highcharts-theme" exampleName="BulletChart" />
 
+## Box plot chart
```

Comment:
What is determining the order of these?

Reply:
Honestly I would just make it alphabetical, any other order is more subjective than we probably need to be. Happy for other opinions though